package.json

```diff
@@ -1573,6 +1573,12 @@
             }
           }
         },
+        "cmake.additionalKitsFile": {
```

Comment:
>additionalKitsFile [](start = 15, length = 18)

Let's make it `additionalKitsFiles` and have it be array of string.

Reply:
Please add `"Items"` property to enforce string for all array items.

---
In reply to: [587658754](https://github.com/Microsoft/vscode-cmake-tools/pull/1701#discussion_r587658754) [](ancestors = 587658754)